package/network/services/odhcpd/files/odhcpd-update

```diff
@@ -1,6 +1,8 @@
 #!/bin/sh
 # Make dnsmasq reread hostfile by sending SIGHUP signal
 
+logger -t odhcpd "*** ODHCPD triggers DNSMASQ reload ***"
```

Comment:
How is this change related to cmake?

Reply:
It isn't. Accidental inclusion in the commit. I will remove it.